Datadog.Trace.sln

```diff
@@ -431,6 +431,14 @@ EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Samples.Shared", "tracer\test\test-applications\Samples.Shared\Samples.Shared.shproj", "{5A806F4B-39E7-4F38-B36F-F5CFC4F8760A}"
 EndProject
 Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
```

Comment:
This was added by VS2022, I can't find a way to get rid of it, but I checked and it doesn't seem to cause issues when opening with VS2019.